packages/components/src/internal/components/auditlog/utils.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('getAuditQueries', () => {
         const auditQueries = getAuditQueries();
         expect(auditQueries.length).toBe(12);
         expect(auditQueries.findIndex(entry => entry == ASSAY_AUDIT_QUERY)).toBeGreaterThanOrEqual(0);
-        expect(auditQueries.findIndex(entry => entry.value === 'inventoryauditevent')).toBe(5);
+        expect(auditQueries.findIndex(entry => entry.value === 'inventoryauditevent')).toBe(10);
```

Comment:
what made these change?

Reply:
They events are in alphabetical order and inventory is now labeled as "storage management events" instead of "freezer management events".